### DISCUSSION TITLE
How to deploy on Digital Ocean serveur with Nginx and PM2?

### DISCUSSION THREAD

**Discussion by MissimahouAKPONON** [calcom/docker#266] [General]
How to deploy on Digital Ocean serveur with Nginx and PM2?

I don't know how to run pm2 with my server

**Comment by krumware** [comment]
Those are not necessarily tools related to running cal.com with Docker. Do you intend on using docker? If not, your question might get better answers on the calcom/cal.com repo

**Reply by MissimahouAKPONON** [reply]
OKay how to deploy it with docker??
